src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -393,5 +393,9 @@
   "stsjorbsmod:FlameWard": {
     "NAME": "Flame Ward",
     "DESCRIPTION": "The next time you are attacked, apply !M! stsjorbsmod:Burning to ALL attackers and gain !B! Block. Exhaust."
+  },
+  "stsjorbsmod:PurgeMind": {
+    "NAME": "Purge Mind",
+    "DESCRIPTION": "Exhaust all cards which \"remember\". NL Deal !stsjorbsmod:BaseDamage! for each card Exhausted. Exhaust. NL Snap."
```

Comment:
I think this description is a little off from the effect; it deals the damage even for cards which were already exhausted.

Reply:
Verified on discord that it's okay to reorder it from the spreadsheet description as "exhaust then damage" rather than "damage then exhaust" if that helps make the card text shorter, though